mmdet/models/dense_heads/anchor_head.py

```diff
@@ -23,7 +23,11 @@ class AnchorHead(BaseDenseHead, BBoxTestMixin):
         anchor_generator (dict): Config dict for anchor generator
         bbox_coder (dict): Config of bounding box coder.
         reg_decoded_bbox (bool): If true, the regression loss would be
-            applied on decoded bounding boxes. Default: False
+            applied directly on decoded bounding boxes, meaning both the
```

Comment:
`meaning both the predicted boxes and regression targets are with absolute` -> `converting both the predicted boxes and regression targets to absolute xxx`